### DISCUSSION TITLE
Is it possible to get type checking for ocamllex .mll files?

### DISCUSSION THREAD

**Discussion by anentropic** [ocamllabs/vscode-ocaml-platform#863] [Ideas]
When working on a .mll file I don't get any of the between-lines type hints (Code Lens?) that I get when editing a regular .ml file

Would be really handy if that was possible

**Comment by rgrinberg** [comment]
It would indeed be handy. But this plugin is completely dependent on merlin to provide this functionality, so there's not much we can do. We can ping @let-def so that he perhaps can keep this issue in the back of his mind meanwhile.

**Reply by let-def** [reply]
I have a POC implementation of ocamllex support in this branch https://github.com/ocaml/merlin/tree/lex-n-yacc
I would like to add menhir/yacc support at some point in the future... But I have no idea when I will have the time.

**Reply by anentropic** [reply]
oh that's great news 🎉

I only need ocamllex... :)
